uqcsbot/scripts/advent.py

```diff
@@ -160,13 +156,26 @@ def format_member(i: int, m: Member):
         return f'{i:>3}) {m.score:>4} {stars} {m.name}'
 
     left = ' ' * (3 + 2 + 4 + 1)  # chars before stars start
-    header = (
-        f'{left}         1111111111222222\n'
-        f'{left}1234567890123456789012345\n'
-    )
+    header = (f'{left}         1111111111222222\n'
+              f'{left}1234567890123456789012345\n')
+
+    return header + '\n'.join(format_member(i, m) for i, m in enumerate(members, 1))
```

Comment:
I didn't know enumerate() took a start parameter 😅

Reply:
How does one go about resolving this comment?

Reply:
I am now aware of the parameter.